app/manager-frontend/src/helpers/dates.js

```diff
@@ -1,6 +1,60 @@
-export const getDateDaysAgo = (days) => {
+import { getDate } from "date-fns";
+
+export const getTimeHoursAgo = () => {
   // gets the date a specified amount of days ago
   var d = new Date();
-  d.setDate(d.getDate() - days);
+  d.setHours(d.getHours() - 2);
   return d;
 };
+
+const getDateString = (date, thisMonth = false, thisYear = false) => {
+  const day = date.getDate();
+  const month = date.toLocaleString("default", { month: "long" }); // get month as string e.g "September"
+  const year = date.getFullYear();
+  const hours = date.getHours();
+  let minutes = date.getMinutes();
+
+  if (minutes < 10) {
+    //add leading 0 to minutes
+    minutes = `0${minutes}`;
+  }
+
+  if (thisMonth) {
+    return `${thisMonth} at ${hours}:${minutes}`;
+  }
+
+  if (thisYear) {
+    return `${month} ${day}  at ${hours}:${minutes}`;
+  }
+  return `${month} ${day} ${year}. at ${hours}:${minutes}`;
```

Comment:
I think you should base the display of the date on the locale of the browser. So it would appear MM-DD-YYYY in en-US but DD-MM-YYYY in en-GB for instance.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
Replaced it with `const dateString = date.toLocaleString("default", options);` which I believe uses the local date of the browser.
Shows like this now:
![Screenshot 2022-10-18 at 09 48 36](https://user-images.githubusercontent.com/75254435/196385692-b4b42839-e89c-4857-a25e-78c566064c83.png)